targets/TARGET_ARM_SSG/mbed_rtx.h

```diff
@@ -22,9 +22,6 @@
 #ifndef INITIAL_SP
 #define INITIAL_SP              (0x20020000UL)
 #endif
-#ifndef OS_TASKCNT
-#define OS_TASKCNT              7
-#endif
 #ifndef OS_MAINSTKSIZE
 #define OS_MAINSTKSIZE          112
```

Comment:
I had to keep initial_sp for now, I don't think there's a portable way of getting it out of linker scripts, needs more investigation I would say post this PR.